anvio/docs/artifacts/functions.md

```diff
@@ -0,0 +1,11 @@
+This artifact **contains information about the functions of the contigs in your %(contigs-db)s.**
+
+It is less a separate file, and more an extension of your %(contigs-db)s that contains this information. 
+
+To get one of these for your %(contigs-db)s, you can either import it (using %(anvi-import-functions)s) or make one yourself by running your contigs against one of two databases available in anvi'o:
+* NCBI [COGs database](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC102395/) -- see %(anvi-run-ncbi-cogs)s for instructions
+* EBI's [Pfam database](https://pfam.xfam.org/) -- see %(anvi-run-pfams)s for instructions
```

Comment:
I think `anvi-run-kegg-kofams` should be in this list, too. It also updates the gene functions table with annotations (in fact, it is essentially the same code as `anvi-run-pfams` except that it uses the KOfam database (which is downloadable from https://www.genome.jp/tools/kofamkoala/ or more specifically ftp://ftp.genome.jp/pub/db/kofam/)

Reply:
I guess that means that functions should be in the provides list for `anvi-run-kegg-kofams` 🤔

Reply:
Done :)